test/versioned/otel-bridge/span.test.js

```diff
@@ -353,3 +357,51 @@ test('fallback metrics are bridged correctly', (t, end) => {
     end()
   })
 })
+
+test('messaging consumer metrics are bridged correctly', (t, end) => {
+  const { agent, tracer } = t.nr
+  const attributes = {
+    [ATTR_MESSAGING_SYSTEM]: 'kafka',
+    [ATTR_MESSAGING_OPERATION]: 'getMessage',
+    [ATTR_SERVER_ADDRESS]: '127.0.0.1',
+    [ATTR_MESSAGING_DESTINATION]: 'input-queue',
+    [ATTR_MESSAGING_DESTINATION_KIND]: 'queue'
+  }
+  const expectedHost = agent.config.getHostnameSafe('127.0.0.1')
+
+  helper.runInTransaction(agent, () => {
+    tracer.startActiveSpan('consumer-test', { kind: otel.SpanKind.CONSUMER, attributes }, (span) => {
+      const tx = agent.getTransaction()
+      const segment = agent.tracer.getSegment()
+      assert.equal(segment.name, 'OtherTransaction/Message/kafka/queue/Named/input-queue')
+      span.end()
+      const duration = hrTimeToMilliseconds(span.duration)
+      assert.equal(duration, segment.getDurationInMillis())
+      agent.getTransaction()
+      tx.end()
+
+      assert.equal(tx.type, 'message')
+
+      const attrs = segment.getAttributes()
+      assert.equal(attrs.host, expectedHost)
+      assert.equal(attrs['messaging.destination'], 'input-queue')
+      assert.equal(attrs['messaging.destination_kind'], 'queue')
+      assert.equal(attrs['messaging.operation'], 'getMessage')
+      assert.equal(attrs['messaging.system'], 'kafka')
+
+      const unscopedMetrics = tx.metrics.unscoped
+      const expectedMetrics = [
+        'OtherTransaction/all',
+        'OtherTransaction/Message/all',
+        'OtherTransaction/Message/kafka/queue/Named/input-queue',
+        'OtherTransactionTotalTime',
+        'OtherTransactionTotalTime/Message/null'
```

Comment:
this is null because we're doing the wrong thing when naming the transaction in `lib/otel/segments/consumer.js`. I'll highlight what needs to change then this name will not have null